src/cpu/gemm/gemm.cpp

```diff
@@ -192,6 +196,25 @@ dnnl_status_t gemm_s8x8s32(const char *transa, const char *transb,
                 B, LDB, bo, beta, C, LDC, co, false);
 #endif
 
+#ifdef DNNL_PPC64
```

Comment:
```suggestion
#if DNNL_PPC64
```
These macros are all defined. Only one of them is set to `1` at a time.

Reply:
I made that change, as it looked like it might fix some of our bugs.  It didn't, but anyway, it's now in there.